Resources/Prototypes/_Goobstation/Heretic/Entities/Mobs/NPC/eldritch_mobs.yml

```
@@ -129,106 +129,4 @@
-    - CannotSuicide
-    - DoorBumpOpener
-  - type: ActiveRadio
-    channels:
-    - Mansus
-  - type: IntrinsicRadioReceiver
-  - type: IntrinsicRadioTransmitter
-    channels:
-    - Mansus
-  - type: Examiner
-  - type: Pullable
+    totalHealth: 100
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)